map_model/src/make/traffic_signals.rs

```diff
@@ -329,6 +332,426 @@ fn stage_per_road(map: &Map, i: IntersectionID) -> ControlTrafficSignal {
     ts
 }
 
+///
+/// Create a traffic signal which has a stage that is: protected straight, protected right,
+/// unprotected left, unprotected right on red. Followed by a variable stage that has protected
+/// left, unprotected right on red. With a last stage that is all-walk and variable.
+/// In some degenerate cases, usually with one or more one-way, this can reduce to stage per road.
+/// In some rare cases, usually with an alleyway, oncoming lanes can't both be protected left turns.
+/// In such cases the stage is split into two stages with each having a protected and yeild turn.
+fn lagging_green(map: &Map, id: IntersectionID) -> Option<ControlTrafficSignal> {
+    let mut ts = new(id, map);
+    if let Some(other) = three_way_three_stage(map, id) {
+        println!("using built-in 3-way");
```

Comment:
Remove all `println!`, or if they'd be helpful in the future, make them `debug!` so they only show up if you run with env variables: https://a-b-street.github.io/docs/dev/index.html#logging

Reply:
oops Thought I got all of these. I'm planning on removing, however I'm also going to look at debug!, as I want to make changes to variable stage handling which currently does event logging.